pkg/contracts/test/RegistryTest.t.sol

```diff
@@ -226,6 +228,47 @@ contract RegistryTest is Test, AlloSetup, RegistrySetupFull, CVStrategyHelpers,
         );
     }
 
+    function test_activate_deactivate_totalActivatedPoints_fixed_system() public {
+        vm.startPrank(pool_admin());
+        uint256 poolId = createPool(
+            allo(),
+            address(strategy),
+            address(_registryCommunity()),
+            registry(),
+            NATIVE,
+            StrategyStruct.ProposalType(0),
+            StrategyStruct.PointSystem.Fixed
+        );
+        console.log("PoolId: %s", poolId);
+        vm.stopPrank();
+
+        vm.startPrank(address(councilSafe));
+        _registryCommunity().addStrategy(address(strategy));
+        vm.stopPrank();
+
+        vm.startPrank(gardenMember);
+        token.approve(address(registryCommunity), STAKE_WITH_FEES);
+        _registryCommunity().stakeAndRegisterMember();
+        //vm.expectRevert("error");
+        strategy.activatePoints();
+
+        // token.approve(address(registryCommunity), tokenAmount * DECIMALS);
+        // _registryCommunity().increasePower(tokenAmount * DECIMALS);
+
+        assertEq(token.balanceOf(address(registryCommunity)), MINIMUM_STAKE, "balance");
+        token.approve(address(registryCommunity), 20 * DECIMALS);
+        _registryCommunity().increasePower(20 * DECIMALS);
+        strategy.deactivatePoints();
+        strategy.activatePoints();
+        vm.stopPrank();
+
+        assertEq(
+            registryCommunity.getMemberPowerInStrategy(gardenMember, address(strategy)),
+            registryCommunity.registerStakeAmount(),
+            "memberPower"
+        );
+    }
```

Comment:
need one more assertion to the value that should be there, it means 50 and also i think increasePower shuold revert and not allow be called in first place, so it means creating new 2 tests for revert too

Reply:
the getter returns 0 for fixed system, I thought it was better like that if not we have to add a bunch of ifs in the higher level (registry contract) so it doesnt call when its fixed because it will revert.

Reply:
what func will revert?